### PR TITLE
Update quick start to use pantheon 1.2 privacy features

### PR DESCRIPTION
**Pull request by antonydenyer**
Placeholder update for post 1.2 launch

### CODE REVIEW

File: privacy/.env [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 # the version of Pantheon Docker image to use.
 # see https://hub.docker.com/r/pegasyseng/pantheon/tags for possible values
-PANTHEON_VERSION=1.1.4
+PANTHEON_VERSION=1.2.0
 ORION_VERSION=0.1.3
```
Comment (josh-richardson) [inline privacy/.env]: Do we not need to use Orion version 1.3.0?

**Comment by NicolasMassart** [comment]
I made a pre-validation against updated web3-eeajs and it LGTM.

File: privacy/.env (hunk 1 above)
Comment (NicolasMassart) [reply privacy/.env]: As we don't use an Orion official Docker image yet, this value is not very useful and honestly you could put any value in it. So don't bother too much with that and we will update the quickstart very soon to have something cleaner, I promise.

**Comment by iikirilov** [comment]
What else needs to happen to get this merged?

**Comment by NicolasMassart** [comment]
Nothing, the release had to happen, so now we can test with the new Pantheon Docker image and then merge. I will do that.

**Comment by iikirilov** [comment]
[PAN-2997] - for completeness

**Pull request merged**